fronts-client/package.json

```diff
@@ -140,12 +132,14 @@
     "styled-components": "5.3.11",
     "ts-optchain": "^0.1.2",
     "tti-polyfill": "^0.2.2",
-    "uuid": "^3.2.1"
+    "url": "^0.11.3",
```

Comment:
Somehow this was missing from our dependencies, but the webpack configuration provided it (it's imported by application code) – this was confusing, but we should certainly have an explicit dependency here.